_episodes/06-style-guide.md

```diff
@@ -3,13 +3,23 @@ title: "Style Guide"
 teaching: 10
 exercises: 0
 questions:
+- "What style should be used for Carpentries lessons?"
 - "How are keyboard key combinations written?"
+- "How are short spans of code written?"
 objectives:
 - "Explain keyboard key combination."
+- "Locate documentation for general matters of style in The Carpentries."
+- "Identify uses of the span of code Markdown feature."
 keypoints:
-- "Keyboard keys need to use `<kbd>` HTML tag."
+- "The Carpentries Handbook contains a style guide."
+- "Keyboard actions are described using the `<kbd>` HTML tag."
+- "The backtick (_\`_) is used as a span of code indicator to enclose code, filenames, and 
```

Comment:
I just wanted to note that, for some reason, an invitation to act (`Within narrative text, enclose short spans of code, filenames, and text to type in backticks.`) to me seems easier to understand than a description of backtick (`The backtick is used as a span of code indicator to`).

Reply:
Across most lessons, the learning objectives are action statements and the key points are phrased as descriptions or factual statements (~ answers to the questions). So for more consistent use of tone that indicates the different function of key points relative to learning objectives, I would prefer to not make it an instruction.  

However, a more active verb construct is easier to understand, so I will rephrase to that end.